tools/dep_graph.sh

```diff
@@ -37,14 +37,14 @@ fi
     {
       cd "$source_dir" || exit 1
       grep -rH --include="*.c" --include="*.h" "^#include \""
-      if test "x$include_libs" != "x"; then
+      if [ $include_libs -gt 0 ]; then
```

Comment:
There is some inconsistency here. I think you need to do this:
```suggestion
      if [ "$include_libs" -ne 0 ]; then
```